libraries/chef_package_helper.rb

```diff
@@ -43,6 +43,7 @@ def install_product(resource)
     #
     #
     def reconfigure_product(resource)
+      install_mixlib_versioning
       chef_ingredient resource.product_name do
         action :reconfigure
```

Comment:
This shouldn't [be necessary](https://github.com/chef-cookbooks/chef-ingredient/blob/master/libraries/chef_ingredient_provider.rb#L65-L66), though?